.travis.yml

```diff
@@ -9,6 +9,7 @@ os:
 julia:
   - 1.0
   - 1.3
```

Comment:
```suggestion
```
to save build resources for the whole organization...
But we would then also need to change the docs build to v1.4